hpke/algs.go

```diff
@@ -31,6 +31,19 @@ const (
 	DHKemX448HkdfSha512 KemID = 0x21
 )
 
+func (k KemID) IsValid() bool {
```

Comment:
These functions are useful to implement `func (suite Suite) isValid() bool`

Reply:
Done.